diff -ur PostNuke-0.762/html/modules/Admin/pnadmin.php PostNuke-0.762.config/html/modules/Admin/pnadmin.php
--- PostNuke-0.762/html/modules/Admin/pnadmin.php	2006-03-10 01:09:47.000000000 +0200
+++ PostNuke-0.762.config/html/modules/Admin/pnadmin.php	2006-03-10 01:09:02.000000000 +0200
@@ -948,8 +948,8 @@
     $config_php = (bool) is_writable('config.php');
     $pnRender->assign('config_php', $config_php);
 
-    // check for config-old.php beeing writable
-    $config_old_php = (bool) is_writable('config-old.php');
+    // check for /etc/webapps/postnuke/config.php.bak beeing writable
+    $config_old_php = (bool) is_writable('/etc/webapps/postnuke/config.php.bak');
     $pnRender->assign('config_old_php', $config_old_php);
 
     // check for .htaccess in /var/lib/postnuke/pnTemp
diff -ur PostNuke-0.762/html/install/check.php PostNuke-0.762.config/html/install/check.php
--- PostNuke-0.762/html/install/check.php	2006-03-10 01:12:07.000000000 +0200
+++ PostNuke-0.762.config/html/install/check.php	2006-03-10 01:11:34.000000000 +0200
@@ -63,7 +63,7 @@
         $chmod = 1;
     } 
 
-    $file = 'config-old.php'; 
+    $file = '/etc/webapps/postnuke/config.php.bak'; 
     // $mode = fileperms($file);
     // $mode &= 0x1ff; # Remove the bits we don't need
     // $chmod = sprintf("%o", $mode);
diff -ur PostNuke-0.762/html/install/modify_config.php PostNuke-0.762.config/html/install/modify_config.php
--- PostNuke-0.762/html/install/modify_config.php	2006-03-10 01:12:06.000000000 +0200
+++ PostNuke-0.762.config/html/install/modify_config.php	2006-03-10 01:11:34.000000000 +0200
@@ -144,7 +144,7 @@
         add_src_rep("adminmail", $email);
     }
 
-    $ret = modify_file("config.php", "config-old.php", $reg_src, $reg_rep);
+    $ret = modify_file("config.php", "/etc/webapps/postnuke/config.php.bak", $reg_src, $reg_rep);
 
     if (preg_match("/Error/", $ret)) {
         show_error_info();
